system/i18n/filetypes.properties

```diff
@@ -36,7 +38,9 @@ ppam.picker.label=Document: ppam
 pptm.picker.label=Document: pptm
 potm.picker.label=Document: potm
 ppsm.picker.label=Document: ppsm
+webp.picker.label=Document: webp
```

Comment:
Should `webp` not be video?

Also, this should be added to `/system/config/Config.cfc$_getConfiguredFileTypes()`.

Reply:
`webp` is an image. https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types.
Updated i18n and added it into config.

Reply:
I also found and fixed typo `serveAsAttachmemnt` of svg mime type in config.